cli/kent/main.go

```
@@ -387,6 +400,235 @@ func runSubcommand(args []string) int {
+func parseCLILiveSessionID(raw string) (runtimeids.SessionID, error) {
+	return runtimeids.ParseSessionID(raw)
+}
+
+func rejectSelfTarget(targetSessionID runtimeids.SessionID, commandText string) error {
+	current, ok := sessionenv.LookupSessionID(os.LookupEnv)
+	if !ok {
+		return nil
+	}
+	currentID, err := parseCLILiveSessionID(current)
+	if err != nil {
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 622) but it returns nil

(nilerr)